src/_base/harness/attributes/common.yml

```diff
@@ -157,6 +157,10 @@ attributes.default:
     default_branch: = @('git.main_branch')
     # potentially deprecated, unless it is repurposed
     main_branch: develop
+  
+  lighthouse:
+    target:
+      url: ~
```

Comment:
Should the default be `= @('hostname')` to pick up on the main URL for the project?

Reply:
Not sure. What do you think @dcole-inviqa ?

Reply:
seems like a sensible default :)